Samples/PhotoEditor/cpp-winui-packaged/PhotoEditor/readme.txt

```diff
@@ -0,0 +1,27 @@
+========================================================================
```

Comment:
Can probably delete this file, I think this comes by default when creating a new C++ WinAppSDK project.

Reply:
Fixed (deleted)